.github/workflows/publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+
+jobs:
+  test:
```

Comment:
This tests it again, separately from the `main.yml` workflow so that we can make sure we only publish test passed code